assets/training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/predict.py

```diff
@@ -202,6 +203,16 @@ def predict(
     logging.info(f"Using device: {device} for the inference")
 
     result = []
+
+    pipe = pipeline(
+        task=task,
+        model=model,
+        tokenizer=tokenizer.tokenizer,
+        feature_extractor=tokenizer.feature_extractor,
+        chunk_length_s=30,
+        device=device,
```

Comment:
The pipeline accepts other parameters too.. Is there a way the user can pass these inputs to the pipeline? You can pass all the params passed via the "kwargs" (pop the values above and send the remaining to the model.) You can also maintain a list of acceptable parameters by the AutomaticSpeechRecognitionPipeline and pass only those values (if they exist in the kwargs) to the pipeline. [Huggingface Automatic Speech Recognition Pipeline doc](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#transformers.AutomaticSpeechRecognitionPipeline)